src/app/pages/homepage/homepage.scss

```diff
@@ -1,26 +1,43 @@
 // The margin between two sections
-$margin-promotion-sections: 60px;
+$margin-promotion-sections: 50px;
 $margin-promotion-sections-small: 15px;
 
 .docs-header-background {
   overflow: hidden;
+  position: relative;
+}
+
+.docs-header-background::before {
+  content: " ";
```

Comment:
I think it can just be: `content: '';` (empty string, no space). Also use single quote